CHANGELOG.md

```diff
@@ -132,6 +138,54 @@ All other Sass mixins, functions and variables prefixed `govuk` are now prefixed
 
 For more information, see [pull request #1289: Deprecate Sass `govuk` namespace usage](https://github.com/nhsuk/nhsuk-frontend/pull/1289).
 
+#### Importing Sass using `all` files
+
+If you're importing the core NHS.UK frontend Sass only, or individual Sass layers (e.g. settings, tools), then you should remove `/all` from each path.
+
+Before:
+
+```scss
+// Example 1: NHS.UK frontend core
+@import "node_modules/nhsuk-frontend/packages/core/all";
+
+// Example 2: NHS.UK frontend layers
+@import "node_modules/nhsuk-frontend/packages/core/settings/all";
+@import "node_modules/nhsuk-frontend/packages/core/tools/all";
+```
+
+After:
+
+```scss
+// Example 1: NHS.UK frontend core
+@import "node_modules/nhsuk-frontend/packages/core";
+
+// Example 2: NHS.UK frontend layers
+@import "node_modules/nhsuk-frontend/packages/core/settings";
+@import "node_modules/nhsuk-frontend/packages/core/tools";
+```
+
+#### Importing Sass individual component partials
+
+If you're importing individual component Sass partials, then you should remove the duplicate component name from each path:
+
+Before:
+
+```scss
+// Example: NHS.UK frontend individual components
+@import "node_modules/nhsuk-frontend/packages/components/components/action-link/action-link";
+@import "node_modules/nhsuk-frontend/packages/components/components/back-link/back-link";
+@import "node_modules/nhsuk-frontend/packages/components/components/breadcrumb/breadcrumb";
+```
+
+After:
+
+```scss
+// Example: NHS.UK frontend individual components
+@import "node_modules/nhsuk-frontend/packages/components/components/action-link";
+@import "node_modules/nhsuk-frontend/packages/components/components/back-link";
+@import "node_modules/nhsuk-frontend/packages/components/components/breadcrumb";
```

Comment:
Not sure that 'components' should be listed twice, as `components/components` - is that a typo?